signer/src/network/libp2p/network.rs

```diff
@@ -224,10 +225,10 @@ mod tests {
     async fn connected_peers_gossip_to_one_another() {
         clear_env();
 
-        let mut rng = get_rng();
-        let key1 = PrivateKey::new(&mut rng);
-        let key2 = PrivateKey::new(&mut rng);
-        let key3 = PrivateKey::new(&mut rng);
+        let rng = &mut get_rng();
```

Comment:
Interesting, I'm half surprised that this compiled.

Reply:
After looking at it I feel like it would be nice to rewrite all tests in this style =)